core/tests/webdriverio_utils/DiagnosticTestPage.js

```diff
@@ -131,6 +122,30 @@ var DiagnosticTestPage = function () {
       expect(recommendedTopicSummaryTiles.length).toEqual(0);
     }
   };
+
+  var AutocompleteTopicDropdownEditor = async function (elem, topicName) {
+    var containerLocator = '.e2e-test-classroom-category-dropdown';
+    var searchInputLocator = '.mat-select-search-input.mat-input-element';
+    var searchInputLocatorTextElement = function (text) {
+      return $$(`.e2e-test-classroom-topic-selector-choice=${text}`);
+    };
+
+    await action.click('Container Element', elem.$(containerLocator));
+    await action.waitForAutosave();
+    // NOTE: the input field is top-level in the DOM, and is outside the
+    // context of 'elem'. The 'select2-dropdown' id is assigned to the input
```

Comment:
Where do we use the fact that the `select2-dropdown` id gets assigned to the activated input field?

Reply:
I have updated code we don't need to pass any element to this function as it's specifically for classroom admin page's topic dropdown.